website/source/docs/guides/creating-certificates.html.md

```diff
@@ -32,140 +32,141 @@ signed by this CA will be allowed to communicate with the cluster.
+
+```shell
+$ consul tls cert create -cli
+==> Using consul-ca.pem and consul-ca-key.pem
+==> Saved consul-cli-0.pem
+==> Saved consul-cli-0-key.pem
 ```
 
+### Note on SANs for Server and Client Certificates
+
 Using `localhost` and `127.0.0.1` as subject alternate names (SANs) allows
 tools like `curl` to be able to communicate with Consul's HTTP API when run on
 the same host. Other SANs may be added including a DNS resolvable hostname to
 allow remote HTTP requests from third party tools.
 
+### What goes where
+
 You should now have the following files:
 
-* `cfssl.json` - cfssl configuration.
-* `consul-ca.csr` - CA signing request.
-* `consul-ca-key.pem` - CA private key. Keep safe!
 * `consul-ca.pem` - CA public certificate.
-* `cli.csr` - Consul CLI certificate signing request.
-* `cli-key.pem` - Consul CLI private key.
-* `cli.pem` - Consul CLI certificate.
-* `client.csr` - Consul client node certificate signing request for the `global` region.
-* `client-key.pem` - Consul client node private key for the `global` region.
-* `client.pem` - Consul client node public certificate for the `global` region.
-* `server.csr` - Consul server node certificate signing request for the `global` region.
-* `server-key.pem` - Consul server node private key for the `global` region.
-* `server.pem` - Consul server node public certificate for the `global` region.
-
-Each Consul node should have the appropriate key (`-key.pem`) and certificate
-(`.pem`) file for its region and role. In addition each node needs the CA's
-public certificate (`consul-ca.pem`).
-
-Please note you will need the keys for the CLI if you choose to disable
-HTTP (in which case running the command `consul members` will return an error).
-This is because the Consul CLI defaults to communicating via HTTP instead of
-HTTPS. We can configure the local Consul client to connect using TLS and specify
-our custom keys and certificates using the command line:
+* `consul-ca-key.pem` - CA private key. Keep safe!
+* `consul-server-dc1-0.pem` - Consul server node public certificate for the `dc1` datacenter.
+* `consul-server-dc1-0-key.pem` - Consul server node private key for the `dc1` datacenter.
+* `consul-client-0.pem` - Consul client node public certificate.
+* `consul-client-0-key.pem` - Consul client node private key.
+* `consul-cli-0.pem` - Consul CLI certificate.
+* `consul-cli-0-key.pem` - Consul CLI private key.
+
+Here is a config for a server:
+
+```json
+{
+  "verify_incoming": true,
+  "verify_outgoing": true,
+  "verify_server_hostname": true,
+  "ca_file": "consul-ca.pem",
+  "cert_file": "consul-server-dc1-0.pem",
+  "key_file": "consul-server-dc1-0-key.pem"
+}
+```
+
+And a config for a client:
+
+```json
+{
+  "verify_incoming": true,
+  "verify_outgoing": true,
+  "ca_file": "consul-ca.pem",
+  "cert_file": "consul-client-0.pem",
+  "key_file": "consul-client-0-key.pem"
+}
+```
+
+Now you need to copy the CA to every machine, the server certificate to the
```

Comment:
This is really good information! I think we could move it up in the guide or even make it step 4. If we make it step 4, I recommend adding a little more detail and separate each item in the list (sentence one) into separate sentences. The "CA private key" could even be its own formatted note.

Reply:
I did that with the new set of steps!